src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -0,0 +1,27 @@
+(function(global, doc, eZ, $) {
+    const parse = () => {
+        const tooltipsNode = doc.querySelectorAll('[title]');
+
+        for (tooltipNode of tooltipsNode) {
+            if (tooltipNode.title.length) {
+                const delay = {
+                    show: tooltipNode.getAttribute('data-delay-show') || 150,
+                    hide: tooltipNode.getAttribute('data-delay-hide') || 75,
+                };
+                const extraClasses = tooltipNode.getAttribute('data-extra-classes') || '';
```

Comment:
Probably could be replaced by `tooltipNode.dataset.extraClasses`.  Ref. https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes